CONTRIBUTING.md

```diff
@@ -84,6 +84,22 @@ To contribute a patch, the workflow is as follows:
   1. Create topic branch
   1. Commit patches
 
+For GUI related issues or pull requests, the https://github.com/bitcoin-core/gui repository should be used.
+For all other issues and pull requests, the https://github.com/bitcoin/bitcoin main repository should be used.
+
+As a rule of thumb, everything that only modifies `src/qt` is a gui-only pull request. However,
+* Global refactoring or other global changes that touch GUI code, as well as
+  non-GUI code, the main repository should be used.
+* For GUI related build system changes, the main repository should be used because
+  the change needs review by the build systems reviewers.
+* Changes in `src/interfaces` need to go to the main repository because they
```

Comment:
these three bullet points should begin with lowercased words and have consistent EOL punctuation